apps/glusterfs/app_cluster.go

```diff
@@ -46,6 +46,11 @@ func (a *App) ClusterCreate(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if err := a.allocator.AddCluster(entry.Info.Id); err != nil {
+		// TODO: lol, error handling
```

Comment:
What is the point of this comment? Why here and not in other places? And why the "lol"?